integrations/operator/CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+## Contributing
+
+### Adding support for new Teleport types
+
+The steps to add support for new Teleport types to the operator are as follows:
+
+#### Make sure the existing CRDs are up to date
+
+In a clean repo before making any changes:
+
+1. Run `make manifests`.
+2. Run `make -C crdgen update-protos`.
+3. Run `make -C crdgen update-snapshot`.
+
+Make sure everything looks sane and commit any changes (there will be some if
+other .proto files used to generate the CRDs have changed).
+
+#### Generate the new CRD
+
+1. Add the type name to the `resources` list in `crdgen/main.go`.
+2. Add the proto file to the `PROTOS` list in `Makefile` if it is not
+   already present. Also add it to the `PROTOS` list in `crdgen/Makefile`.
+3. Run `make manifests` to generate the CRD.
```

Comment:
Add a comment for people to check if the crd change make sense and is compatible with teleport YAML representation otherwise they need to handle the custom case